Archane.js

```diff
@@ -0,0 +1,1169 @@
+            playerPerception = rollDieTwenty() + playerWisdomStatModifier(playerWisdom)
+            console.log(chalk`You roll a {red ${playerPerception}}! {cyan (Wisdom Modifier of ${playerWisdomStatModifier(playerWisdom)})}`)
+            if (playerPerception >= 11) {
+                console.log(`\nYou were able to scope the area and find three doors down the hallway.`)
+                threeDoors();
+            } else {
+                console.log(`\nYou tried scoping the area, however, you were unable to find anything.
+In this case, you decided to go with your second plan. Pretend that you are bound until the guard finds you.`)
+                console.log(chalk`{red Roll for Deception!}`)
+                playerDeceptionRoll = rollDieTwenty() + playerCharismaStatModifier(playerCharisma)
+                console.log(chalk`You roll a {red ${playerDeceptionRoll}} {cyan (Charisma Modifier of ${playerCharismaStatModifier(playerCharisma)})}`)
+                if (playerDeceptionRoll >= 11) {
+                    console.log(`\nYou decide to go back to the room you were held in.
+You then wrap the rope back around you, but not tight so that you have room to escape if needed.
+You wait until the guard comes.`)
+                    guardAppearance();
+                } else {
+                    console.log(`\nYou decide to go back to the room you were held in.
+You then wrap the rope back around you, but it is obvious that the guard isn't going to believe you.`)
+                    guardAppearance();
+                }
+            }
+        } else {
+            console.log(chalk`\nYou try to free yourself from the rope.
+{red You end up giving yourself a cramp in your arm.}
+You also unintentonally made the rope tighter around you.`)
+            guardAppearance();
+        }
+    } else if (playerChairChoices === "impression") {
+        console.log(chalk`\n{red Roll for Deception!}`)
+        playerDeceptionRoll = rollDieTwenty() + playerCharismaStatModifier(playerCharisma)
+        console.log(chalk`You roll a {red ${playerDeceptionRoll}} {cyan (Charisma Modifier of ${playerCharismaStatModifier(playerCharisma)})}`)
+        if (playerDeceptionRoll >= 11) {
+            console.log(chalk`\nYou were able to loosen up the rope around you, giving the {magenta impression} that you are still tied up but are able to slip out at any time.`)
+            guardAppearance();
+        } else {
+            console.log(chalk`\nYou try to free yourself from the rope.
+{red You end up giving yourself a cramp in your arm.}
+You also unintentonally made the rope tighter around you.`)
+            guardAppearance();
+        }
+    } else if (playerChairChoices === "stats") {
+        logPlayerStats();
+        playerChairChoice();
+    } else if (playerChairChoice === "inventory") {
+        logplayerInventory();
+        playerChairChoice();
+    } else {
+        playerChairChoice();
+    }
+}
+
+const threeDoors = () => {
```

Comment:
This function is very long. Consider breaking up into smaller helper functions. Maybe a function for each door choice.